firewood/src/storage/linear/historical.rs

```diff
@@ -11,12 +11,22 @@ use super::{LinearStore, ReadLinearStore};
 /// A [Historical] [LinearStore] supports read operations only
 #[derive(Debug)]
 pub(crate) struct Historical<P: ReadLinearStore> {
-    old: BTreeMap<u64, Box<[u8]>>,
-    parent: Arc<LinearStore<P>>,
+    pub(crate) old: BTreeMap<u64, Box<[u8]>>,
```

Comment:
This means we bleed the implementation details of Historical outside this file. Why do we need to do that?